Shared/CountdownApp.swift

```diff
@@ -11,7 +11,10 @@ import SwiftUI
 struct CountdownApp: App {
```

Comment:
Não sou familiar com esse conceito de App, como funciona ? Vi que não tem appdelegate ?

Reply:
Na verdade o `AppDelegate` teve suas responsabilidades divididas com o `SceneDelegate` a partir do Xcode 11. Isso aconteceu pois no iOS 13 a apple introduziu o conceito de multi-window para iPadOS e MacOS e com isso resolveu que `AppDelegate` cuida do eventos do app e o `SceneDelegate` cuida do eventos da UI. O SwiftUI não conseguia ser utilizado sem depender do UIKit, por isso aquela "gabiarra" com o `UIHostingController`.
Agora na WWDC20, eles apresentaram sua nova `Black Magic` 😁 , o protocolo `App`, que é gerado quando vc inicia um projeto `SwiftUI App`. Ele cuida da vida da UI, apresentando a View dentro de um `Scene` e gerenciando o ciclo do sistema através de um `@Environment(\.scenePhase)`, que por ser um `keypath` pode ser observado com o modificador `onChange(of:)`.

Acho que consegui resumir 😀